src/components/button-cta/button-cta.tsx

```diff
@@ -15,13 +15,14 @@ import React, { FC, ReactNode } from "react";
 import ButtonSize from "../button/button-size";
 import ButtonTheme from "../button/button-theme";
 import ButtonCtaNavigationMethod from "./button-cta-navigation-method";
-import Target from "../target/target.model";
+import { Target } from "../target/target.model";
 
 // Styles
 import { button, large, primary, secondary } from "./button-cta.module.css";
 
 interface Props {
   attributeHREF: string;
+  attributeTarget?: Target;
```

Comment:
Added target an an optional prop to ButtaCTA component - allows the "Explore Datasets" button on the homepage to open in the same tab.